src/annotator/config/index.js

```diff
@@ -162,8 +198,33 @@ export function getConfig(appContext = 'annotator', window_ = window) {
   let filteredKeys = configurationKeys(appContext);
   filteredKeys.forEach(name => {
     const configDef = configDefinitions[name];
+    const hasDefault = configDef.defaultValue !== undefined; // A default could be null
+    const isURLFromBrowserExtension = isBrowserExtension(
```

Comment:
I was thinking to rename the `isBrowserExtension` function rather than the local variable here, so as to use the shorter name for the most local symbol. Also `isURLFromBrowserExtension` makes more sense as the function name because it tests the provided URL, whereas the local variable here means "is the client being loaded from the browser extension".

Reply:
Awe, I totally misread that :)
**will change**